src/activities/associateGrade/GradeCategoryEntity.js

```diff
@@ -0,0 +1,13 @@
+import { Entity } from '../../es6/Entity';
+
+/**
+ * GradeCategoryEntity class representation of a grade category
+ */
+export class GradeCategoryEntity extends Entity {
```

Comment:
Could we use the `GradeCategoryEntity` in `src/activities` instead of a new one that does the same thing. I think they'll always be the same.

Reply:
They are the same, my thinking is instead of having two locations for these related entities we would put this into the `associateGrade` folder, then when we are no longer using the original one we can remove it from `src/activities` (I guess we don't have to wait, we can just have activities use the new location right away).